src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/idam/SearchEventAndCase.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.idam;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(
+    value       = "SearchEventAndCase",
+    description = "Search task request with event and case id"
+)
+@SuppressWarnings({"PMD.UnnecessaryAnnotationValueElement"})
```

Comment:
I believe you can fix this by removing the JsonProperty from either the field or from the getMethod. I don't think you need both.